web/frontend/src/api/slam.ts

```diff
@@ -74,3 +74,19 @@ export const getSLAMPosition = async (robotClient: Client, name: string) => {
 
   return response?.getPose();
 };
+
+export const getSLAMMapInfo = async (robotClient: Client, name: string) => {
```

Comment:
Can we call this `getLatestMapInfo`? The function names in this folder usually match the function name they wrap